__tests__/with-urql-client.spec.tsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import { shallow, configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import { NextFC } from 'next';
+import { NextComponentType } from 'next/types';
```

Comment:
I think we should be able to just `import` this directly from `next`. They should be moving their types to a top-level lib export:

```suggestion
import { NextComponentType } from 'next';
```